src/components/elements/HighlightTable/HighlightTable.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import styled from 'styled-components';
+import { TableBody, TableCell, TableHeader, TableRow } from '../Table';
+import { colorPrimary } from '../../../styles/colors';
+
+const TableStyle = styled.table`
```

Comment:
We're repeating a lot css here. We create a base `TableStyled` styled-component and extend it as necessary by doing `const HighlightTableStyle = styled(TableStyle)`